app/controllers/api/pictures_controller.rb

```diff
@@ -1,11 +1,26 @@
 module Api
   class PicturesController < BaseController
     def create_resource(_type, _id, data)
+      validate_picture_data(data)
```

Comment:
@imtayadeway I did, actually. I actually looked into validating the actual content type to ensure it is a picture via Paperclip (they have built in validation for attachments that I thought I could leverage since we already use it) and some other methods. It wasn't that straight forward though, so I felt like I was going down a rabbit hole. However if you think that's the way I should go, I can spend more time on it.